mssqlcli/locale/mssql-cli.pot

```diff
@@ -0,0 +1,130 @@
+# Translations template for PROJECT.
```

Comment:
Why are we checking in the .pot? Isn't this a template to generate the .po files?

Reply:
I followed how azdata did. You can see azdata repository contains *.pot files generated by pybabel command.

Reply:
Removed pot from commit